commands/group_test.go

```diff
@@ -166,3 +166,99 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
 		s.Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestTeamGroupStatusCmd() {
+	s.Run("Should fail when team is not found", func() {
+		printer.Clean()
+
+		teamID := "teamId"
+		arg := teamID
+		args := []string{arg}
+		cmd := &cobra.Command{}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamID, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(teamID, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := teamGroupStatusCmdF(s.client, cmd, args)
+
+		s.Require().EqualError(err, "Unable to find team '"+args[0]+"'")
+	})
+
+	s.Run("Should show valid response when status of team group status is not present", func() {
+		printer.Clean()
+
+		teamID := "teamId"
+		arg := teamID
+		args := []string{arg}
+		cmd := &cobra.Command{}
+		team := &model.Team{Id: teamID}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamID, "").
+			Return(team, &model.Response{Error: nil}).
+			Times(1)
+
+		err := teamGroupStatusCmdF(s.client, cmd, args)
+
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], "Disabled")
+	})
+
+	s.Run("Should show valid response when status of team group status is enabled", func() {
```

Comment:
Here I would use:  
`Should show valid response when group constraints status for a team is enabled`